samples/grids/hierarchical-grid/data-performance-virtualization/src/RemoteService.ts

```diff
@@ -0,0 +1,23 @@
+const URL = `https://services.odata.org/V4/Northwind/Northwind.svc/`;
```

Comment:
Similar to the case in [this PR for WC](https://github.com/IgniteUI/igniteui-wc-examples/pull/683#pullrequestreview-1979913935), the URL is broken and no data is loaded.